oms-active-directory-security-audit-solution/azuredeploy.parameters.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "value": "GEN-UNIQUE-10"
+        },
+        
+        "workspaceRegionId": {

```

Comment:
remove this and use a defaultValue in the template

Reply:
can you explain bit more on what to remove

Reply:
done